packages/icons/web/v2/cafe_24.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you check with your designer to make sure that the 24px versions for both the `cafe` and the `hospitality` icons are also added to the Figma icons library?

I can see larger, two-color variants (not supported by `@sumup/icons` yet) in there, but not these versions. ([internal Figma link](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC/SumUp-Iconography?node-id=8050%3A4701))